packages/clay-management-toolbar/src/ClayResultsBar.js

```diff
@@ -37,6 +37,9 @@ class ClayResultsBar extends ClayComponent {
 	 */
 	_handleFilterLabelCloseClicked(event) {
 		return !this.emit({
+			data: {
+				label: this.filterLabels[event.target.data.labelId],
```

Comment:
I think we will likely need more than the label value, as that can be localized or who know what else. We would be better off passing down the `id`, the whole list of `label` and any additional information we might need.

Reply:
That is already the entire label object.